test/new/db/tools/rabin2

```diff
@@ -417,8 +417,7 @@ RUN
 NAME=rabin2 -g
 FILE=../bins/elf/analysis/x86-helloworld-gcc
 CMDS=!rabin2 -g ${R2_FILE}
-EXPECT=<<EOF
-[Sections]
+EXPECT="[Sections]
```

Comment:
Did you change <<eof in " manually or was it r2r that did it?

Reply:
r2r did that.

Reply:
uhm we should use <<EOF , the "" syntax is deprecated in r2r.v

Reply:
Changed it back :)